cdp_scrapers/instances/seattle.py

```diff
@@ -72,14 +72,11 @@ def __init__(self):
         )
 
         try:
+            url = "https://seattlechannel.org/"
             urlopen("https://seattlechannel.org/")
-        except URLError:
-            pass
-        else:
-            raise Exception(
-                "seattlechannel.org may have fixed their SSL cert. "
-                "Check and fix 'requests.get(*, verify=False)' calls"
-            )
+        except URLError as ue:
+            log.error(f"Unable to reach {url}", ue)
+            return
```

Comment:
We should still raise here, because connection error to SeattleChannel is a requirement. i.e. Our event gathering pipeline requires meeting videos ([`IngestionModel.Session` requires `video_uri`](https://github.com/CouncilDataProject/cdp-backend/blob/92b0484c0df80af4a4026489f554c5d86aefb1e4/cdp_backend/pipeline/ingestion_models.py#L149)). So, there is no point in continuing.